### PR TITLE
Dependabot oppdateringer 20210902-175306

### PR DESCRIPTION
**Pull request by blommish**
Erstatter:
 - https://github.com/navikt/familie-ef-sak-frontend/pull/467
 - https://github.com/navikt/familie-ef-sak-frontend/pull/466
 - https://github.com/navikt/familie-ef-sak-frontend/pull/465
 - https://github.com/navikt/familie-ef-sak-frontend/pull/464
 - https://github.com/navikt/familie-ef-sak-frontend/pull/463
 - https://github.com/navikt/familie-ef-sak-frontend/pull/461
 - https://github.com/navikt/familie-ef-sak-frontend/pull/459

Merget inn etterpå
- https://github.com/navikt/familie-ef-sak-frontend/pull/462

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -104,12 +104,12 @@
...
     "nav-frontend-spinner": "^2.0.17",
     "nav-frontend-spinner-style": "^1.0.2",
-    "nav-frontend-tabell-style": "^2.0.2",
+    "nav-frontend-tabell-style": "^2.1.1",
     "nav-frontend-tabs": "^2.0.1",
     "nav-frontend-tabs-style": "^2.0.1",
     "nav-frontend-typografi": "^3.1.1",
     "nav-frontend-typografi-style": "^1.1.1",
-    "node-fetch": "^2.6.1",
+    "node-fetch": "^3.0.0",
```
Comment (charliemidtlyng) [inline package.json:112]: Trengte vi ikke å gjøre noen endringer for å oppdatere denne?
https://github.com/node-fetch/node-fetch/blob/HEAD/docs/v3-UPGRADE-GUIDE.md

**Review by charliemidtlyng — APPROVED** [review]

**Comment by ma10s** [comment]
Når jeg kjører start:dev lokalt får jeg denne feilen:
tsc && NODE_ENV=development node node_dist/backend/server.js
{"level":"info","message":"Environment variable HTTP_PROXY is not set, not proxying requests for openid-client"}
internal/modules/cjs/loader.js:1080
      throw new ERR_REQUIRE_ESM(filename, parentPath, packageJsonPath);

start:mock fungerer fint. Noen tips?

**Comment by eirarset** [comment]
@blommish Har du testet dette? Jeg har en branch liggende med samme oppdateringene, pluss til til, og sliter med en runtime feil.

**Comment by blommish** [comment]
Fikset i en annen PR

**Pull request closed without merging**